packages/contracts/test/core/dao/dao.ts

```diff
@@ -1210,22 +1209,17 @@ describe('DAO', function () {
     });
 
     it('should call the signature validator', async () => {
-      const ERC1271MockFactory = await smock.mock('ERC1271Mock');
-      const erc1271Mock = await ERC1271MockFactory.deploy();
+      const magicValue = '0x41424344';
 
-      await dao.setSignatureValidator(erc1271Mock.address);
-      await dao.isValidSignature(ethers.utils.keccak256('0x00'), '0x00');
-      expect(erc1271Mock.isValidSignature).has.been.callCount(1);
-    });
+      const erc1271Mock = await hre.wrapper.deploy('ERC1271Mock');
 
-    it('should return the validators response', async () => {
-      const ERC1271MockFactory = new ERC1271Mock__factory(signers[0]);
-      const erc1271Mock = await ERC1271MockFactory.deploy();
+      await erc1271Mock.setMagicValue('0x41424344');
```

Comment:
```suggestion
      await erc1271Mock.setMagicValue('magicValue');
```
See also https://github.com/aragon/osx/pull/596/files#r1602801671

Reply:
Not sure how this is correct. The mock contract must expect bytes4. Check ERC1271Mock contract above.

Reply:
I'd argue better to have these as const imports

Reply:
sorry, fixed the suggestion.
Just use the local var in the scope containing the same value instead of duplicating it.